cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -415,6 +415,26 @@ func ValidateMixedArguments(flag *pflag.FlagSet) error {
 	return nil
 }
 
+func isAllowedFlag(flagName string) bool {
+	isAllowed := false
+	switch flagName {
+	case kubeadmcmdoptions.CfgPath,
```

Comment:
How about putting these into a Set and check whether flagName is contained in the Set ?

Reply:
our flags are kind of sparse and cover different functionality i.e. the flags don't make sense to be a part of the same set outside of this validatation, but i guess we can consider it for a future refactor.

Reply:
I have a patch ready, should I send out PR ?

Reply:
given the flags are mostly unrelated, my initial reaction is, no.
but you can still send if you want to get feedback on the change.

Reply:
See #74032